### PR TITLE
CLOUDP-67836: Add a function to handle output formats

### PR DESCRIPTION
**Pull request by gssbzn**
## Proposed changes

_Jira ticket:_ CLOUDP-67836

## Checklist

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

## Further comments

This adds a function to centralise the output management and changing all commands using the json formatter to use this function instead

### CODE REVIEW

File: internal/output/output.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+type Config interface {
+	Output() string
+}
+
+const jsonFormat = "json"
+
+// Print outputs v to os.Stdout while handling configured formats,
+// if the optional t is given then it's processed as a go-template
+func Print(c Config, t string, v interface{}) error {
```
Comment (gssbzn) [inline internal/output/output.go:32]: this is the only change that matters, the rest is changing all commands to use this

File: internal/output/output.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+	if c.Output() == jsonFormat {
+		return json.PrettyPrint(v)
+	}
+	if t != "" {
+		tmpl, err := template.New("test").Parse(t)
```
Comment (robcarlan-mlab) [inline internal/output/output.go:37]: Should the name be something like "output"?

Comment (gssbzn) [reply internal/output/output.go:37]: indeed

**Comment by gssbzn** [comment]
@robcarlan-mlab merged master and updated your last command to use the new func

**Pull request merged**